src/schema/addons.schema.ts

```diff
@@ -1,9 +1,22 @@
-import { modelOptions } from '@typegoose/typegoose';
+import { ApiProperty } from '@nestjs/swagger';
+import { Prop, modelOptions } from '@typegoose/typegoose';
 import { Types } from 'mongoose';
 
 @modelOptions({
   schemaOptions: { collection: 'addons' },
 })
 export class AddonSchema {
   _id: Types.ObjectId;
```

Comment:
Sai swagger decorators hai field nee duay krub kuay

```typescript
  @Prop({ auto: true })
  @ApiProperty({ type: String, description: 'Session ID' })
  _id: Types.ObjectId;
```

Reply:
do i need to do this with category too??

Reply:
btw what does the description: 'Session ID' do

Reply:
> do i need to do this with category too??

Yes, for every api property

Reply:
> btw what does the description: 'Session ID' do

For a sexy schema documentation (well it's what @hirasawaau wrote)
 
![image](https://github.com/meeorder/meeorder-backend/assets/56222561/5d635fe1-34db-4760-9f84-e8b089afe25c)

Reply:
FYI

![image](https://github.com/meeorder/meeorder-backend/assets/56222561/8e5e6e6e-5ac2-4b58-88e4-58840fab28d4)